resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -762,6 +813,11 @@ public class ApiClient implements AutoCloseable {
             return this;
         }
 
+         public Builder withMaxRetryTimeInSec(int maxRetryTimeInSec) {
```

Comment:
In preparation for unit testing, it would be a good pattern to expand this concept to accept a RetryConfiguration object instead of a single value. These values can be passed into the Retry object's constructor. The RetryConfiguration object would contain settings for:

* backoffInterval - defaults to 5 minutes, replaces FIVE_MINUTES_IN_MILLIS
* defaultDelay - defaults to 3 seconds. replaces BASE_BACKOFF_TIME_IN_SEC
* maxRetryTime - defaults to 0, replaces existing config option

This will allow the unit tests to validate that the backoff logic works on a much shorter timescale.